site/src/modules/dashboard/Navbar/UserDropdown/UserDropdown.tsx

```diff
@@ -19,6 +20,8 @@ export interface UserDropdownProps {
   onSignOut: () => void;
 }
 
+export const accessibleDropdownLabel = "Open dropdown menu for user options";
```

Comment:
off: I'm always wondering how long labels should be. Imagining myself using an interface via audio...  long texts could slow me down. Just a random thought.